lib/DDG/Spice/RenegoJobSearch.pm

```diff
@@ -0,0 +1,32 @@
+package DDG::Spice::RenegoJobSearch;
+
+# ABSTRACT: Renego Job Search
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://services.renego.net/jobs/ws.php?action=getjobs&sid=3&partnerid=43&what=$1&signkey={{ENV{DDG_SPICE_RENEGO_APIKEY}}}&blocksize=10&orderby=1&radius=25&offset=0&where=$2&fields=inserted|title|description_short|host|url_its|inserted|job_company_name|location_name&rt=json';
+spice from => '(.*)/(.*)';
+
+triggers any => 'jobs', 'job';
+
+primary_example_queries "manager job berlin";
+secondary_example_queries "manager jobs";
+description "Renego Job Search";
+name "RenegoJobSearch";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/RenegoJobSearch.pm";
+topics "special_interest";
+attribution github => ['https://github.com/mightycid','Sebastian Szeracki'];
+
+# Handle statement
+handle remainder => sub {
+
+    return $_, $loc->city if $_;
```

Comment:
@mightycid in order to limit this to Germany, we'll need to check that `$loc` is defined and the `country` is Germany.

Take a look at https://github.com/duckduckgo/zeroclickinfo-spice/pull/2307/files to learn how to check for the proper Locale.

As well, please take a look at how they create tests to ensure it triggers appropriately. We should have a few tests that ensure it works when it should, and that it skips queries that shouldn't trigger it (e.g. from the wrong local)